prefix/none.go

```diff
@@ -12,7 +12,7 @@ func (p *nonePrefixer) EncodeLength(int, int) ([]byte, error) {
 }
 
 func (p *nonePrefixer) DecodeLength(fixLen int, data []byte) (int, int, error) {
-	return len(data), 0, nil
+	return fixLen, 0, nil
```

Comment:
`None` prefixer is used in cases when you want to work with the value of the field and when you don't know the length upfront. If you know the length upfront - just use `ASCII.Fixed`.

Let's revert this change.

Reply:
done